Index: src/QuizkampenClient.java
===================================================================
diff --git a/src/QuizkampenClient.java b/src/QuizkampenClient.java
--- a/src/QuizkampenClient.java	
+++ b/src/QuizkampenClient.java	
@@ -4,12 +4,13 @@
 import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+
 import java.awt.image.BufferedImage;
 import java.io.*;
 import java.net.InetAddress;
 import java.net.Socket;
 
-public class QuizkampenClient implements ActionListener {
+public class QuizkampenClient implements ActionListener{
     JFrame frame = new JFrame("Quizkampen");
     JPanel basePanel = new JPanel();
     BufferedImage backgroundImage = ImageIO.read(new File("background.jpg"));
@@ -18,10 +19,10 @@
     JLabel score = new JLabel("Poäng: ");
     JTextField nameField = new JTextField("", 10);
     JButton play = new JButton("Starta spel");
-    JButton category1 = new JButton("Film");
-    JButton category2 = new JButton("Musik");
-    JButton category3 = new JButton("Java-kunskap");
-    JButton category4 = new JButton("Övrigt");
+    JButton button1 = new JButton("");
+    JButton button2 = new JButton("");
+    JButton button3 = new JButton("");
+    JButton button4 = new JButton("");
     Answers answers = new Answers();
 
     InetAddress ip = InetAddress.getLocalHost();
@@ -33,6 +34,7 @@
     String currentQuestion;
 
     public QuizkampenClient() throws IOException {
+
         frame.setContentPane(new JLabel(new ImageIcon(backgroundImage)));
         frame.setLayout(new FlowLayout());
         frame.add(basePanel);
@@ -54,13 +56,13 @@
         score.setFont(new Font("Tahoma", Font.PLAIN, 23));
 
         String serverResponse;
-        while(true) {
-            if ((serverResponse = in.readLine()) != null){
+        while (true) {
+            if ((serverResponse = in.readLine()) != null) {
                 title2.setText(serverResponse);
                 frame.revalidate();
                 frame.repaint();
                 basePanel.add(play);
-                play.addActionListener(e ->out.println("startPressed"));
+                play.addActionListener(e -> out.println("startPressed"));
                 instructionsPlayer();
                 break;
             }
@@ -68,28 +70,29 @@
     }
 
     public void instructionsPlayer() throws IOException {
-        while(true){
+        while (true) {
             String inFromServer = in.readLine();
-            if(inFromServer.equals("SET CATEGORY")){
+            if (inFromServer.equals("SET CATEGORY")) {
                 title.setText("Välj kategori");
                 basePanel.remove(title2);
                 basePanel.remove(nameField);
                 basePanel.remove(play);
                 basePanel.add(score);
-                basePanel.add(category1);
-                basePanel.add(category2);
-                basePanel.add(category3);
-                basePanel.add(category4);
-                category1.addActionListener(e -> out.println("Film"));
-                category2.addActionListener(e -> out.println("Musik"));
-                category3.addActionListener(e -> out.println("Java-kunskap"));
-                category4.addActionListener(e -> out.println("Övrigt"));
+                button1.setText("Film");
+                basePanel.add(button1);
+                basePanel.add(button2);
+                basePanel.add(button3);
+                basePanel.add(button4);
+                button1.addActionListener(e -> out.println("Film"));
+                button2.addActionListener(e -> out.println("Musik"));
+                button3.addActionListener(e -> out.println("Java-kunskap"));
+                button4.addActionListener(e -> out.println("Övrigt"));
                 frame.repaint();
                 frame.revalidate();
                 answerQuestion();
                 break;
             }
-            if(inFromServer.equals("SET WAIT")){
+            if (inFromServer.equals("SET WAIT")) {
                 title.setText("Väntar på din tur..");
                 basePanel.remove(title2);
                 basePanel.remove(nameField);
@@ -103,29 +106,17 @@
     }
 
     private void answerQuestion() throws IOException {
-        JButton answer1 = new JButton("Svarsalternativ 1");
-        JButton answer2 = new JButton("Svarsalternativ 2");
-        JButton answer3 = new JButton("Svarsalternativ 3");
-        JButton answer4 = new JButton("Svarsalternativ 4");
-        while(true){
+
+        while (true) {
             String questionAsked = in.readLine();
-            int answerIndex = Integer.parseInt(in.readLine()+1);
-            if(questionAsked != null){
+            int answerIndex = Integer.parseInt(in.readLine() + 1);
+            if (questionAsked != null) {
                 title.setText(questionAsked);
-                basePanel.add(answer1);
-                basePanel.add(answer2);
-                basePanel.add(answer3);
-                basePanel.add(answer4);
 
-                answer1.setText(answers.filmAnswers.get(answerIndex-1).get(0));
-                answer2.setText(answers.filmAnswers.get(answerIndex-1).get(1));
-                answer3.setText(answers.filmAnswers.get(answerIndex-1).get(2));
-                answer4.setText(answers.filmAnswers.get(answerIndex-1).get(3));
-
-                basePanel.remove(category1);
-                basePanel.remove(category2);
-                basePanel.remove(category3);
-                basePanel.remove(category4);
+                button1.setText(answers.filmAnswers.get(answerIndex - 1).get(0));
+                button2.setText(answers.filmAnswers.get(answerIndex - 1).get(1));
+                button3.setText(answers.filmAnswers.get(answerIndex - 1).get(2));
+                button4.setText(answers.filmAnswers.get(answerIndex - 1).get(3));
 
                 frame.repaint();
                 frame.revalidate();
@@ -137,11 +128,16 @@
     public static void main(String[] args) throws IOException {
         new QuizkampenClient();
     }
+
     @Override
     public void actionPerformed(ActionEvent e) {
         out.println(nameField.getText());
         title2.setText("Väntar på en motståndare..");
         frame.repaint();
         frame.revalidate();
+
+
     }
+
+
 }
